<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QMainWindow" name="SimulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="simulationCountSpinBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartSimulation">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск симуляції</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultsTable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>691</width>
      <height>271</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Номер симуляції</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Результат</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
